inc/admin/metabox/main.php

```diff
@@ -142,13 +125,8 @@ private function add_content_toggles() {
 	 * Add content width control.
 	 */
 	private function add_content_width() {
-
-		$enabled_default = ( self::is_new_page() || self::is_checkout() ) ? 'on' : 'off';
-		$width_default   = ( self::is_new_page() || self::is_checkout() ) ? 100 : 70;
-		if ( $this->new_skin ) {
-			$enabled_default = 'off';
-			$width_default   = self::is_post() ? 70 : 100;
-		}
+		$enabled_default = 'off';
```

Comment:
I'm not sure why this was changed. The condition should remain the same:
`$enabled_default = ( self::is_new_page() || self::is_checkout() ) ? 'on' : 'off';`